### DISCUSSION TITLE
Presets of toggles

### DISCUSSION THREAD

**Discussion by Perseus333** [Pedro-Gregorio/TubeMod#49] [Ideas]
This would be something more in the long-run when more features are enabled but I think that having some predefined presets or even letting the users create them. The presets would be a combination of toggles or settings that would fit a certain use case. There could be some like: ultra minimalism, no distractions, clean layout, etc. This in conjunction with #48 would remove friction when customizing the layout.

I know that this is not the priority for now, and that it might be overly complicated, but it was just to store the idea, because otherwise I'd forget lol.

**Comment by Pedro-Gregorio** [comment]
Alright, time to come back to this! Presets are coming (probably) on 1.10.0 for the following categories:
- Shorts
- Ads

Any idea regarding how you would like it implemented, UX wise?

**Reply by Perseus333** [reply]
Maybe like a separate tab? You know, in one tab you would have all the buttons for fined tunned toggling and on the other tab you would have the presets? And if you select one preset it just automatically toggles all of the individual buttons on the other tab, but you can then toggle each one of them on and off regardless.  I will try to provide an illustration today to better explain myself.

**Reply by Pedro-Gregorio** [reply]
With a separate tab you mean like the one we have for Header, Sidebar, etc.? 🤔

Edit: Also, yes, the goal of the presets is to toggle a couple of individual things at once. There's some extra logic to this, to properly showcase the state, but it's doable. Had a simple 'No Shorts' preset locally 😄
